dracoon/src/main/java/ch/cyberduck/core/sds/SDSSession.java

```diff
@@ -256,7 +256,7 @@ public void process(final HttpRequest request, final HttpContext context) {
                 .register(new JSON())
                 .register(JacksonFeature.class)
                 .connectorProvider(new HttpComponentsProvider(apache))));
-        final int timeout = preferences.getInteger("connection.timeout.seconds") * 1000;
+        final int timeout = ConnectionTimeoutFactory.get().getTimeout() * 1000;
```

Comment:
Should HostPreferences be used here?

Reply:
We can by default refactor this to `HostPreferences` instead of directly accessing Preferences.

Reply:
Didn't see preferences already is HostPreferences-instance.